test/cypress/integration/reusable/check-sdk-space-methods-test.ts

```diff
@@ -1,8 +1,15 @@
-import cachedContentTypes from '../fixtures/cached-content-types'
+import { removeVariableData } from '../../utils/remove-variable-data'
+import expectedContentTypes from '../fixtures/cached-content-types'
 
 export function verifySpaceGetCachedContentTypes(iframeSelector: string) {
-  cy.getSdk(iframeSelector).then(sdk => {
-    expect(sdk.space.getCachedContentTypes()).to.deep.equal(cachedContentTypes)
+  cy.getSdk(iframeSelector).then((sdk) => {
+    const cachedContentTypes = sdk.space.getCachedContentTypes()
+
+    for (const ct of cachedContentTypes) {
```

Comment:
An empty array would make this test succeed. Can't we do

```typescript
expect(sdk.space.getCachedContentTypes().map(removeVariableData)).to.deep.equal(expectedContentTypes.map(removeVariableData));
```

Reply:
Good point. Maybe it's more readable if we iterated on the `expectedContentTypes` (which is fixed and always present), without nesting `map`s. WDYT?

Reply:
yes that would also work. I agree that this nesting is a little confusing. Of course we could also extract the value within `expect(...)` and `equal(...)` to it's own variable, but I actually don't care.
In case of an error many small expects probably provide a better diff.

Reply:
> In case of an error many small expects probably provide a better diff.

This was exactly the reasoning :)

Reply:
Then just iterate over `expectedContentTypes` 👍